RoadPricingAngular/src/app/registration/registration.component.html

```diff
@@ -0,0 +1,35 @@
+<app-header></app-header>
+<div class="container">
+  <div class="col-xs-12 col-md-8">
+    <h1>{{ 'REGISTER.TITLE' | translate }}</h1>
+    <form #userForm="ngForm">
+      <div class="form-group">
+        <label for="name">{{ 'REGISTER.NAME_LABEL' | translate }}</label>
+        <input [(ngModel)]="owner.name" id="name" type="text" class="form-control input-sm" name="name" required/>
```

Comment:
De input velden hoeven niet two-way gebind te worden, alleen one-way zodat de dingen die je typt in de invoervelden in owner property van je component worden opgeslagen.